tabbed-mode-explainer.md

```diff
@@ -20,22 +20,21 @@ Add a new manifest field `tab_strip` which allows apps to customize the tab stri
 
 ```
 "tab_strip": {
-    "home_tab": "auto" | "absent" | {
-        "url": "auto" | <url>,
-        "icons": "auto" | [...],
+    "home_tab": "absent" | {
+        "icons": [...],
+        "scope_patterns": [{"pathname": "..."}]
     },
-    "new_tab_button": "auto" | "absent" | {
-        "url": "auto" | <url>,
-        "icons": "auto" | [...],
+    "new_tab_button": "absent" | {
+        "url": <url>,
```

Comment:
```suggestion
        "url": <url>,
        "icons": [...]
```

Reply:
(At least you say so in line 53.)

Reply:
Yeh I wasn't too sure if I should add the potential extension fields in here - same with url for home_tab. Wdyt?

Reply:
I'd add it and mark it somehow as optional.